projects/plugins/jetpack/extensions/shared/register-jetpack-block.native.js

```diff
@@ -7,7 +7,7 @@ import { registerBlockType } from '@wordpress/blocks';
  * Internal dependencies
  */
 import extensionList from '../index.json';
-import getJetpackExtensionAvailability from './get-jetpack-extension-availability';
+import getJetpackExtensionAvailability from '../../../../js-packages/shared-extension-utils/src/get-jetpack-extension-availability';
```

Comment:
Did you try `@automattic/get-jetpack-extension-availability` for the import statement? I wouldn't expect having to explicitly move up four directories to import that file.